addon/webextension/background/selectorLoader.js

```diff
@@ -0,0 +1,74 @@
+/* globals browser, catcher */
+window.selectorLoader = (function () {
+  const exports = {};
+  // These modules are loaded in order, and some need to be listed before others
+  // due to dependencies:
+  const scripts = [
+    "catcher.js",
+    "background/selectorLoader.js",
+    "selector/callBackground.js",
+    "clipboard.js",
+    "makeUuid.js",
+    "build/shot.js",
+    "randomString.js",
+    "domainFromUrl.js",
+    "build/inlineSelectionCss.js",
+    "selector/documentMetadata.js",
+    "selector/util.js",
+    "selector/ui.js",
+    "selector/shooter.js",
+    "selector/uicontrol.js"
+  ];
+
+  exports.unloadIfLoaded = function () {
+    return browser.tabs.executeScript({
+      code: "window.selectorLoader && window.selectorLoader.unloadModules()"
```

Comment:
This replaces the code in `lifecycle.js` that unloaded the previous run. Since the toggle disables on the second press we don't want to load the scripts again after this.